test/e2e/client_test.go

```diff
@@ -22,6 +22,13 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+type Config struct {
+	// Directory to place artifacts when a test fails
+	ArtifactDir string `envconfig:"ARTIFACTS" required:"false"`
```

Comment:
ARTIFACT_DIR is used in our tests, see [here](https://github.com/openshift/release/blob/3cbd3516ac42e15ede15052ce55201afe1aa0079/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L76).